README.md

```diff
@@ -1 +1,7 @@
-# cr-review-project
\ No newline at end of file
+# past-project-trials
+
+
+>  This project is part of the code reviewers training.
+
+## Desktop 💻
+![screenshot](imgs/desktopVersion.png)
```

Comment:
- [ ] Please make sure to add a professional README.md file.

Reply:
- [ ] Good job spotting the need for a good README.md file, but I think you should try to give a bit more explanation on why you are asking for this change. For example, you could better say: `Please make sure to add a professional README.md file. A good README.md file is important to keep your project under good documentation, and also helps everyone that wants to use your project to understand how to set it up or how to use it`